tests/test_plugin_binance_csv.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2022 macanudo527

```

Comment:
Can you rename this file test_plugin_binance_com_supplemental_csv.py? This is to avoid conflict with a potential binance.com CSV reader test.